src/api/app/components/confirmation_dialog_component.html.haml

```diff
@@ -6,7 +6,7 @@
       .modal-body
         %p.confirmation-text= confirmation_text
 
-        = form_tag(action, method: method) do
+        = form_tag(action, method: method, remote: true) do
```

Comment:
The parameters `action` and `method` are passed when we call the component, in other to make this component reusable from other dialogs of the application. I would do the same with `remote` because not all the dialogs are going to use Ajax.

Anyway, this change is not compulsory in this PR. We can change it at the time we are going to re-use the component. We have [a card](https://trello.com/c/gNicWZMY/2578-replace-partials-by-confirmationdialogcomponent) for that in our Free Style backlog.